pkg/appStore/chartGroup/ChartGroupService.go

```diff
@@ -773,11 +775,25 @@ func (impl *ChartGroupServiceImpl) DeployDefaultChartOnCluster(bean *cluster2.Cl
 			chartGroupInstallRequest.UserId = userId
 			var chartGroupInstallChartRequests []*ChartGroupInstallChartRequest
 			for _, item := range charts.ChartComponent {
-				appStoreApplicationVersionId, err := impl.appStoreApplicationVersionRepository.FindLatestAppStoreVersionIdByAppStoreName(item.Name)
+				appStore, err := impl.appStoreRepository.FindAppStoreByName(item.Name)
 				if err != nil {
-					impl.logger.Errorw("DeployDefaultChartOnCluster, error in getting app store", "data", t, "err", err)
+					impl.logger.Errorw("error in getting app store by name", "appStoreName", item.Name, "err", err)
 					return false, err
 				}
+				var appStoreApplicationVersionId int
+				if len(appStore.DockerArtifactStoreId) > 0 {
```

Comment:
can we define a variable like isOCIRepo := `len(appStore.DockerArtifactStoreId) > 0`